src/injector/README.md

```diff
@@ -31,3 +56,41 @@ echo "Linux binary size: $size_linux"
 size_aarch64=$(du --si target/aarch64-unknown-linux-musl/release/zarf-injector | cut -f1)
 echo "aarch64 binary size: $size_aarch64"
 ```
+
+## Testing your injector
+
+Point the [zarf-registry/zarf.yaml](../../packages/zarf-registry/zarf.yaml) to
+the locally built injector image.
+
+```
+    files:
+      # Rust Injector Binary
+      - source: ../../src/injector/target/x86_64-unknown-linux-musl/release/zarf-injector
+        target: "###ZARF_TEMP###/zarf-injector"
+        <!-- shasum: "###ZARF_PKG_TMPL_INJECTOR_AMD64_SHASUM###" -->
+        executable: true
+
+    files:
+      # Rust Injector Binary
+      - source: ../../src/injector/target/aarch64-unknown-linux-musl/release/zarf-injector
+        target: "###ZARF_TEMP###/zarf-injector"
+        <!-- shasum: "###ZARF_PKG_TMPL_INJECTOR_ARM64_SHASUM###" -->
+        executable: true
+```
+
+In Zarf Root Directory, run `make init-package`, if you are running on an Apple Silicon
```

Comment:
Do we need a step to run make from the injector src or is that elsewhere?

Reply:
updated this to use make cmds